src/modules/chatExtras/assets/lightDesignChatHistory.ts

```diff
@@ -0,0 +1,18 @@
+export default (redesignActive: boolean, modalName: string) => {
+    let startPhrase = redesignActive ? `#modals-container [data-modal="${modalName}"] + .vm--modal .redesign-wrapper ` : '';
+    document.querySelectorAll<HTMLSpanElement>(`${startPhrase}div.well span.pull-right`).forEach((el: HTMLSpanElement) => {
+        el.classList.remove('pull-right');
```

Comment:
I would really appreciate if you differentiate between the timestamp and the message